src/FSharp.Formatting.Common/Range.fs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FSharp.Formatting.Common
+
+type MarkdownRange = { StartLine : int; StartColumn : int; EndLine : int; EndColumn : int }
+
+module MRange =
```

Comment:
A slightly more idiomatic thing would be to name the module `MarkdownRange` (same as the type name), which requires the following attribute:

    [<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
    module MarkdownRange =
       (...)